controllers/drplacementcontrol_controller_test.go

```diff
@@ -2581,7 +2581,7 @@ var _ = Describe("DRPlacementControl Reconciler", func() {
 			deleteUserPlacement(UserPlacementName, DefaultDRPCNamespace)
 			deleteDRPC()
 			waitForCompletion("deleted")
-			Expect(getManifestWorkCount(East1ManagedCluster)).Should(Equal(2))       // DRCluster + NS MW only
+			Expect(getManifestWorkCount(East1ManagedCluster)).Should(Equal(1))       // DRCluster + NS MW only
```

Comment:
(note) the tests are failing because the prior DRPC delete does not call `waitForCompletion("deleted")` and then proceed with DRCluster deletion. As a result we have a stale VRG MW on the west1 cluster preventing the next test from passing.

This needs to be addressed in 2 places, [here](https://github.com/RamenDR/ramen/blob/c5c4f7eda8442ca966ee2e05a9b83bfe2cfc39d5/controllers/drplacementcontrol_controller_test.go#L2485) and [here](https://github.com/RamenDR/ramen/blob/c5c4f7eda8442ca966ee2e05a9b83bfe2cfc39d5/controllers/drplacementcontrol_controller_test.go#L2541).